modules/connatixBidAdapter.js

```diff
@@ -61,6 +62,16 @@ export function validateVideo(mediaTypes) {
   return video.context !== ADPOD;
 }
 
+/**
+ * Get ids from Prebid User ID Modules and add them to the payload
+ */
+function _handleEids(payload, validBidRequests) {
+  let bidUserIdAsEids = deepAccess(validBidRequests, '0.userIdAsEids');
+  if (isArray(bidUserIdAsEids) && bidUserIdAsEids.length > 0) {
+    deepSetValue(payload, 'identityProvider', bidUserIdAsEids);
```

Comment:
the naming of this param is a bit confusing 'identityProvider' 🤔, as we're sending a list of idsnot some kind of identity provider. Any chance to change it?

Reply:
sure. changed to `userIdList`